ArcGISRuntimeSDKQt_CppSamples/Geometry/ClipGeometry/ClipGeometry.qml

```diff
@@ -49,4 +49,18 @@ ClipGeometrySample {
             clipAreas();
         }
     }
+
+    Button {
+        id: resetButton
```

Comment:
Consider having this button disabled by default, enabled once clipButton is clicked, then disabled again when resetButton itself is clicked